bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,269 @@
+# Babylon Node Setup
+
+## Table of Contents
+
+1. [Install Babylon Binary](#install-babylon-binary)
+2. [Setup Node Home Directory and Configure](#setup-your-node-home-directory-and-configuration)
+4. [Create Keys](#create-keys)
```

Comment:
Do we need to add the `4. [Create Keys]` and `6. [Get Funds]` steps into this document as both of them are not needed to sync a new node

Reply:
That's a good point, makes sense to move these to the validator guide instead.